pyproject.toml

```diff
@@ -1,9 +1,27 @@
 [build-system]
-requires = ["setuptools", "wheel"]
+
+requires = ["setuptools>=78.1.0", "wheel"]
 build-backend = "setuptools.build_meta"
 
 [tool.isort]
 profile = "black"
 
 [tool.mypy]
 strict = true
+
+[project]
+name = "script-runner"
+version = "0.1.0"
+description = "A web application for running Python scripts."
+readme = "README.md"
+license = { text = "apache2" }
+requires-python = ">=3.12"
+dynamic = ["dependencies"]
+[tool.setuptools.dynamic]
+dependencies = { file = ["requirements.txt"] }
+
+[tool.setuptools.package-data]
+app = ["config.schema.json", "frontend/dist/**/*"]
```

Comment:
the frontend dist directory won't exist when this runs in CI as that folder isn't checked into code, only the source files are. this probably needs a custom build script that runs the relevant npm commands to generate these files